templates/cube/microcerts.html

```diff
@@ -7,16 +7,41 @@
 {% block content %}
 
 {% if user %}
-<section class="p-strip--suru is-dark is-deep">
-  <div class="row u-equal-height">
-    <div class="col-8">
+<section class="p-strip--square-suru has-cube is-dark is-slanted--bottom-left">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-6">
       <h1>Your progress to CUBEdom</h1>
-      <p>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor.
-      </p>
+      <p class="p-heading--4">You have completed {{ passed_courses }} of 15 MicroCertifications</p>
     </div>
-    <div class="col-4"></div>
+    <div class="col-6 p-cube-progression u-hide--medium u-hide--small">
+      <!-- <div class="p-cube-progression is-tag u-align--center u-vertically-center">
+        <span><small>Micro<br /><span>Certifications:</span></small><br /><span class="p-heading--4">{{ passed_courses }}/15</span></span>
+      </div> -->
```

Comment:
As a general rule, we should not check in commented-out code. However, if there is a good reason for it, you should leave a comment above explaining.